code/game/area/areas/ruins/jungle.dm

```diff
@@ -96,3 +96,33 @@
 	name = "cargo dump"
 	icon_state = "dk_yellow"
 
+// Syndicate Shitfort
+
+/area/ruin/jungle/shitfort
```

Comment:
Please rename your areas to not be "shitfort".

Reply:
Done.